gobblin-compaction/src/main/java/gobblin/compaction/dataset/DatasetsFinder.java

```diff
@@ -68,8 +68,8 @@ public DatasetsFinder(State state) {
     this.inputDir = getInputDir();
     this.destDir = getDestDir();
     this.tmpOutputDir = getTmpOutputDir();
-    this.blacklist = getBlacklist();
```

Comment:
Are these changes as well as the ones to `MRCompactor` suppose to be in this PR?